models/group_model.go

```diff
@@ -7,20 +7,18 @@ import (
 
 type Group struct {
 	mgm.DefaultModel `bson:",inline"`
-	Users            []primitive.ObjectID `bson:"users"`
-	Drift            []primitive.ObjectID `bson:"drift"`
+	Name             string               `bson:"name" json:"name" binding:"required"`
+	Users            []primitive.ObjectID `bson:"users" json:"users" binding:"required"`
```

Comment:
Not _completely_ necessary per se, since we have this check in group_controller's create route, but we could potentially add `min=1` to the Users field here as well, just in case (or maybe so in the future, if we support kicking members from a group, we can ensure that a group doesn't get completely nuked to 0 members). 

Regardless, its not a super important issue to handle right now so feel free to add it in if you want.